### PR TITLE
feat: Set domain on asset message FS-318

### PR DESCRIPTION
**Pull request by sb88k**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-318" title="FS-318" target="_blank">Sub-taskFS-318</a>  Set domain on the asset message
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Set domain on asset message

### Dependencies (Optional)

- https://github.com/wireapp/wire-ios-protos/pull/59

### CODE REVIEW

**Review by typfel — APPROVED** [review]

File: Tests/Source/Model/Utils/PBMessageValidationTests.swift [hunk 1]
```diff
@@ -233,43 +233,43 @@ class PBMessageValidationTests: XCTestCase {
     // MARK: Assets
 
     func testThatItCreatesMessageWithValidAsset() {
-        XCTAssertNotNil(genericMessage(assetId: "asset-id", assetToken: "token", preview: true))
-        XCTAssertNotNil(genericMessage(assetId: "asset-id", assetToken: "token=", preview: false))
+        XCTAssertNotNil(genericMessage(assetId: "asset-id", assetToken: "token", assetDomain: "domain", preview: true))
```
Comment (johnxnguyen) [inline Tests/Source/Model/Utils/PBMessageValidationTests.swift:236]: What effect does the domain have in these tests? Is there a distinction between a valid and invalid domain?

Comment (sb88k) [reply Tests/Source/Model/Utils/PBMessageValidationTests.swift:236]: It doesn't really have effect. I see some validation on the id and the token, but there is no validation for the domain.

**Review by johnxnguyen — APPROVED** [review]

**Comment by sb88k** [comment]
release major

**Comment by zenkins** [comment]
Hello, @sb88k! Merging PR for command **release major** for PR #**1304**. Will create a **major** release after merged.
Next version is 283.0.0.

**Comment by sb88k** [comment]
release major

**Comment by zenkins** [comment]
Hello, @sb88k! Merging PR for command **release major** for PR #**1304**. Will create a **major** release after merged.
Next version is 283.0.0.

**Pull request merged**